spec/dummy/package.json

```diff
@@ -9,7 +9,9 @@
   "dependencies": {
     "@babel/preset-react": "^7.10.4",
     "@rails/webpacker": "^5.2.1",
+    "@rescript/react": "^0.10.3",
     "babel-plugin-module-resolver": "^4.0.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
```

Comment:
Is this relevant to ReScript, or is it just unrelated improvement?

Reply:
remove-prop-types is something to save space on production builds.

@gscarv13 why is this being added for rescript?

Reply:
I was building the app for the tests and that was being required. I'll check if it is still relevant @justin808